jetstream/README.md

```diff
@@ -494,6 +494,10 @@ request. An error will be triggered if at least 2 heartbeats are missed
 - `WithConsumeErrHandler(func (ConsumeContext, error))` - when used, sets a
   custom error handler on `Consume()`, allowing e.g. tracking missing
   heartbeats.
+- `PullMaxMessagesWithBytesLimit` - up to the provided number of messages
```

Comment:
do we need to note somewhere that MaxBytes should be bigger than the max payload size from the nats-server? To avoid the getting stuck not being able to receive the next message issue?

Reply:
Yeah, I suppose it's worth to add this, I'll create a new PR with this info